src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/integration/eventManager/GetAllocatedEvents.kt

```diff
@@ -0,0 +1,129 @@
+package uk.gov.justice.digital.hmpps.hmppsworkload.integration.eventManager
+
+import org.junit.jupiter.api.Test
+import uk.gov.justice.digital.hmpps.hmppsworkload.domain.CaseType
+import uk.gov.justice.digital.hmpps.hmppsworkload.domain.Tier
+import uk.gov.justice.digital.hmpps.hmppsworkload.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsworkload.integration.domain.AllocationDetailIntegration
+import uk.gov.justice.digital.hmpps.hmppsworkload.integration.mockserver.WorkforceAllocationsToDeliusExtension.Companion.workforceAllocationsToDelius
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.CaseDetailsEntity
+import uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity.EventManagerEntity
+import java.time.ZoneOffset
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+import java.time.temporal.ChronoUnit
+
+class GetAllocatedEvents : IntegrationTestBase() {
+
+  @Test
+  fun `can get all allocated events by logged in user`() {
+    val loggedInUser = "SOME_USER"
+    val storedEventManager = eventManagerRepository.save(
+      EventManagerEntity(
+        crn = "CRN1",
+        staffCode = "OM1",
+        teamCode = "T1",
+        createdBy = loggedInUser,
+        isActive = true,
+        eventNumber = 2
+      )
+    )
+
+    val caseDetails = caseDetailsRepository.save(CaseDetailsEntity(crn = storedEventManager.crn, tier = Tier.B2, type = CaseType.COMMUNITY, "", ""))
+
+    workforceAllocationsToDelius.allocationDetails(listOf(AllocationDetailIntegration(storedEventManager.crn, storedEventManager.staffCode)))
```

Comment:
I do find this mock access confusing. Without anything indicating it is a mock makes me think it's calling something - like the above line sets up the DB - but it's just setting the mock.